src/08-accessing.js

```diff
@@ -171,7 +195,10 @@ function getFullAddress(person) {}
       numbers: [7185550921, 7185558611],
     };
  */
-function getFlatObject(person) {}
+function getFlatObject(person) {
+  let flatObj = {...person.names, ...person.numbers, ...person.address}
```

Comment:
This is very slick but do we know what exactly is happening under the hood here?  What is the name of the ` ... ` operator and what is it doing?